test/e2e/logforwarding/syslog/forward_to_syslog_test.go

```diff
@@ -139,6 +143,17 @@ var _ = Describe("[ClusterLogForwarder] Forwards logs", func() {
 				Expect(logStore.GrepLogs(grepmsgid, helpers.DefaultWaitForLogsTimeout)).To(Equal(expectedMsgID), "Expected: "+expectedMsgID)
 				expectedProcID := forwarder.Spec.Outputs[0].Syslog.ProcID
 				Expect(logStore.GrepLogs(grepprocid, helpers.DefaultWaitForLogsTimeout)).To(Equal(expectedProcID), "Expected: "+expectedProcID)
+				grepMsgContent := fmt.Sprintf(`grep %s %%s| grep pod_name | tail -n 1 | sed -n -e 's/^.*openshift://p'`, logGenPod)
```

Comment:
please implement the needed method in `syslog.go`:
```
func (syslog *syslogReceiverLogStore) HasApplicationLogs(timeToWait time.Duration) (bool, error) {
	return false, fmt.Errorf("Not implemented")
}
```

Reply:
Why we need to implement this method? The logs generated by log-generator pod while testing is shipped to infra.log. There is no application logs generated during the process.